src/views/selfCodesPanel/index.ts

```diff
@@ -0,0 +1,12 @@
+import * as vscode from 'vscode';
+
+class SelfCodesPanelProvider {
```

Comment:
Is this used anywhere? There is no `export` in this file.

Reply:
just used later in `vscode-db2i.runEditorStatement`